eng/pipelines/common/sdk-insertion.yml

```diff
@@ -57,3 +61,21 @@ jobs:
         inputs:
           artifactName: vsdrop-multitarget-signed
           downloadPath: $(Build.StagingDirectory)/$(MultiTargetVSDropCommitStatusName)
+  - task: DotNetCoreCLI@2
+    displayName: Generate and publish BAR manifest
+    inputs:
+      projects: $(Build.SourcesDirectory)\src\Workload\Microsoft.Maui.Sdk\Microsoft.Maui.Sdk.csproj
+      arguments: >-
+        -t:PushManifestToBuildAssetRegistry
+        -p:BuildAssetRegistryToken=$(MaestroAccessToken)
+        -p:OutputPath=$(Build.StagingDirectory)\nuget\
+        -v:n -bl:$(Build.StagingDirectory)\binlogs\push-bar-manifest.binlog
+    condition: and(succeeded(), eq('${{ parameters.pushMauiPackagesToMaestro }}', 'true'))
+  - powershell: |
+      $versionEndpoint = 'https://maestro-prod.westus2.cloudapp.azure.com/api/assets/darc-version?api-version=2019-01-16'
+      $darcVersion = $(Invoke-WebRequest -Uri $versionEndpoint -UseBasicParsing).Content
+      $arcadeServicesSource = 'https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json'
+      & dotnet tool update microsoft.dotnet.darc --version "$darcVersion" --add-source "$arcadeServicesSource" --tool-path $(Agent.ToolsDirectory)\darc -v n
+      & $(Agent.ToolsDirectory)\darc\darc add-build-to-channel --default-channels --id $(BARBuildId) --publishing-infra-version 3 --skip-assets-publishing --password $(MaestroAccessToken) --azdev-pat $(publishing-dnceng-devdiv-code-r-build-re)
```

Comment:
With these changes in place, we will need to make sure to manually create "default channels" for MAUI for every major branch, and also every time we create a new release/servicing branch:

    darc add-default-channel --channel ".NET 9.0.1xx SDK" --branch "net9.0" --repo https://github.com/dotnet/maui

Reply:
I wrote down this command in a doc like this:

* https://github.com/xamarin/xamarin-android/blob/main/Documentation/guides/HowToBranch.md

Reply:
Yeah right now this is fine.. We can also add to the loop. @PureWeen  knows the correct place for sure. He as looking at organising better our release process. Should it be on the wiki to be public?